presto-main/src/test/java/com/facebook/presto/execution/TestQueryTrackerHighTaskCountKill.java

```diff
@@ -64,4 +70,33 @@ public void testMultipleQueriesKilledDueToTaskCount()
             scheduledExecutorService.shutdownNow();
         }
     }
+
+    @Test
+    public void testMultipleQueriesKilledDueToTaskCount_withClusterQueryTracker()
```

Comment:
only the large query is killed right?
how is this testing multiple queries killed.